dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -2570,15 +2558,14 @@ void collectAllCols(gp_walk_info& gwi, Item_field* ifp)
                 SimpleColumn* sc = new SimpleColumn();
                 sc->columnName(cols[j]->alias());
                 sc->colPosition(j);
-                string tableAlias(csep->derivedTbAlias());
-                sc->tableAlias(lower(tableAlias));
-                sc->viewName(lower(gwi.tbList[i].view));
+                sc->tableAlias(csep->derivedTbAlias());
+                sc->viewName(gwi.tbList[i].view);
```

Comment:
These 2 calls should have the `lower_case_table_names` as the second argument.

Reply:
All values put into csep or gwi have already been converted